infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/UniqueWorkspacePVCStrategy.java

```diff
@@ -141,9 +145,11 @@ public void prepare(KubernetesEnvironment k8sEnv, String workspaceId, long timeo
     LOG.debug("Creating PVCs for workspace '{}'", workspaceId);
     k8sClaims.createIfNotExist(k8sEnv.getPersistentVolumeClaims().values());
 
-    LOG.debug("Waiting PVCs for workspace '{}' to be bound", workspaceId);
-    for (PersistentVolumeClaim pvc : k8sEnv.getPersistentVolumeClaims().values()) {
-      k8sClaims.waitBound(pvc.getMetadata().getName(), timeoutMillis);
+    if (waitBound) {
+      LOG.debug("Waiting PVCs for workspace '{}' to be bound", workspaceId);
```

Comment:
Is it intentional that here it's "Waiting PVCs" plural and in CommonPVCStrategy.java it's just "Waiting PVC" singular?

Reply:
Also, proper English here would be something like "Waiting for workspace PVC(s) {} to be bound"

Reply:
Thanks for catching it.

> Is it intentional that here it's "Waiting PVCs" (multiple PVCs), but in CommonPVCStrategy.java it's just "Waiting PVC" (one singular PVC)?

Yes, it is intentional, because:
- CommonPVCStrategy: one PVC is used for all workspaces;
- PerWorkspace: one PVC is used for one workspace;
- Unique: one PVC is used per one workspace volume, and usually it's more than 2, because we use at least volume for `plugins` and `projects`.

> Waiting for workspace PVC {} to be bound

`{}` here is a workspace id. Then what is the better here:
1. Waiting for workspace PVCs `workspace123oxusd2` to be bound.
2. Waiting for workspace `workspace123oxusd2` PVCs  to be bound.
3. Waiting for PVCs of workspace `workspace123oxusd2`  to be bound.

@nickboldt

Reply:
if in all three cases only one PVC is listed then the (s) seems unnecessary. 

That said, the third option is the most grammatically correct and clear about ownership hierarchy.